packages/runtime/container-runtime/garbageCollection.md

```diff
@@ -1,28 +1,38 @@
+## What do I need to do?
+All Fluid objects that are in use must be marked as referenced so that they are not deleted by GC. Similarly, references to all unused Fluid objects should be removed so that they can be deleted by GC. It is the responsibility of the users of Fluid Framework to correctly add and remove references to Fluid objects.
 
-    `Root` objects can never be deleted so be careful and only create them if they should live forever.
-  - Store a handle ([IFluidHandle](../../../common/lib/core-interfaces/src/handles.ts)) to the object in a referenced DDS that supports handle in its data. For example, a data store's handle can be stored in a referenced `SharedMap` DDS.
-- All references to unused Fluid objects should be removed so that they can be deleted by GC. To remove an object's reference, all its handles should be removed from referenced DDSes.
+## How do I reference / unreference Fluid objects?
+Currently, the only Fluid objects that are eligible for GC are data stores and attachment blobs. The following sections describe how you can mark them as referenced or unreferenced.
+### Data stores
+There are 2 ways to add reference to data stores:
+  - Store the data stores's ([IFluidHandle](../../../common/lib/core-interfaces/src/handles.ts)) in a referenced DDS that supports handle in its data. For example, a data store's handle can be stored in a referenced `SharedMap` DDS.
+  - Alias the data store. Aliased data stores are rooted in the container, i.e., they are always referenced and cannot be unreferenced later. Aliased data stores can never be deleted so only do so if you want them to live forever.
 
-> Note that there should be at least one `root` data store with one or more DDSes in a Fluid document so that other objects' handles can be stored in it.
+To remove reference to a data store, remove all its IFluidHandles from any referenced DDSes. The data store then becomes eligible for GC.
+
+> Note that there should be at least one aliased data store with at least one DDS in a container. This is the starting point for GC to look for other referenced objects in the container.
+
+### Attachment blobs
+The only way to reference attachment blobs is to store its IFluidHandle in a referenced DDS similar to data stores.
+
+To remove reference to an attachment blob, remove all its IFluidHandles from any referenced DDSes. The attachment blob then becomes eligible for GC.
 
 ## GC algorithm
 The GC algorithm runs in two phases:
 
 ### Mark phase
 In this phase, the GC algorithm identifies all Fluid objects that are unreferenced and marks them as such:
-- It starts at the root data stores and marks them, and all their DDSes as referenced.
+1. It starts at the root (aliased) data stores and marks them and all their DDSes as referenced.
     > Note: Currently all DDSes are considered as root so they are always referenced. This may change in the future.
```

Comment:
This phrasing "all DDSes are considered as root" makes sense to you and me (because we're thinking about the symmetry of layers and how they're root _within their containing DataStoreRuntime_), but at first glance it sounds like every DDS in a container is considered root.  So I would avoid this wording.